docs/admin/authorization/abac.md

```diff
@@ -24,7 +24,7 @@ properties:
 
   - Versioning properties:
     - `apiVersion`, type string; valid values are "abac.authorization.kubernetes.io/v1beta1". Allows versioning and conversion of the policy format.
-    - `kind`, type string: valid values are "Policy". Allows versioning and conversion of the policy format.
+    - `kind`, type string; valid values are "Policy". Allows versioning and conversion of the policy format.
```

Comment:
`kind` should be a separate bullet from `apiVersion`.  Your changes to ";" and "." are ok, but kind and apiVersion are now showing up as one bullet point on the page